src/mlpack/methods/ann/layer/separable_convolution.hpp

```diff
@@ -0,0 +1,416 @@
+/**
+ * @file separable_convolution.hpp
+ * @author Kartik Dutt
```

Comment:
I think it would be okay, if you add you here as well.

Reply:
do we add another author with just a `,` or with another `@author` tag ? and i thought that majority of the work was done by @kartikdutt18 so it was better to leave it that way.